content/en/synthetics/private_locations/_index.md

```diff
@@ -179,7 +179,7 @@ Launch your private location on:
 Run this command to boot your private location worker by mounting your configuration file to the container. Ensure that your `<MY_WORKER_CONFIG_FILE_NAME>.json` file is in `/etc/docker`, not the root home folder:
 
 ```shell
-docker run --rm -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetics-check-runner.json datadog/synthetics-private-location-worker:latest
+docker run -d --restart always -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetics-check-runner.json datadog/synthetics-private-location-worker:latest
```

Comment:
I think we can keep the `--rm` flag which cleans up the resources when the container is stopped (the PL is stateless so this should not be an issue).

`unless-stopped` may also be prefered for this use case. The difference is if the container is manually stopped, it won't run when the docker daemon restart, which may be preferable for this kind of image (unlike the agent which is more of a "background container" maybe) ? 

```suggestion
docker run --rm -d --restart unless-stopped -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetics-check-runner.json datadog/synthetics-private-location-worker:latest
```